### ISSUE TITLE
used to work;  doesn't now

### ISSUE THREAD

**Issue by lewisl** [sglyon/PlotlyJS.jl#219]
Possibly some more documentation is needed on how to respond to Blink.AtomShell.install() prompts so that the process doesn't fail.  Can't debug this.

```
julia> Blink.AtomShell.install()
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  386k  100  386k    0     0   327k      0  0:00:01  0:00:01 --:--:--  327k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   619    0   619    0     0   1360      0 --:--:-- --:--:-- --:--:--  1360
100 46.4M  100 46.4M    0     0  3116k      0  0:00:15  0:00:15 --:--:-- 7494k
replace LICENSE? [y]es, [n]o, [A]ll, [N]one, [r]ename: n
replace LICENSES.chromium.html? [y]es, [n]o, [A]ll, [N]one, [r]ename: n
replace version? [y]es, [n]o, [A]ll, [N]one, [r]ename: n
mv: rename Electron.app to Julia.app/Electron.app: Directory not empty
ERROR: failed process: Process(`mv Electron.app Julia.app`, ProcessExited(1)) [1]
Stacktrace:
 [1] error(::String, ::Base.Process, ::String, ::Int64, ::String) at ./error.jl:42
 [2] pipeline_error at ./process.jl:712 [inlined]
 [3] #run#509(::Bool, ::Function, ::Cmd) at ./process.jl:670
 [4] run at ./process.jl:668 [inlined]
 [5] (::getfield(Blink.AtomShell, Symbol("##3#4")))() at /Users/lewis/.julia/packages/Blink/KMJt9/src/AtomShell/install.jl:37
 [6] cd(::getfield(Blink.AtomShell, Symbol("##3#4")), ::String) at ./file.jl:96
 [7] install() at /Users/lewis/.julia/packages/Blink/KMJt9/src/AtomShell/install.jl:27
 [8] top-level scope at none:0
```

**Comment by cc7768** [comment]
Hmmm -- Is this a PlotlyJS or a Blink issue?

**Comment by lewisl** [comment]
I honestly don’t know.  What used to work via the Plots api was the ability to close a plot window created from the Julia REPL in macos terminal.  I’d assumed that there’d be some similar command in the PlotlyJS API.  Spencer requested that I use it directly rather than through Plots until the 0.7/1.0 work was sorted out.  But, there doesn’t  appear to be any such command.  Also, when you do using PlotlyJS in Julia, even though PlotlyJS requires Blink there is no direct programmatic access unless in your own code you “using Blink” directly.  But, this is not document.  I suppose there must be some Blink command to close active windows.  I am looking for something in the plotting API.

**Comment by cc7768** [comment]
Your second comment makes this seem like a duplicate of https://github.com/sglyon/PlotlyJS.jl/issues/220. I'm going to go ahead and close this one in favor of the more specific issue.

If you're having trouble installing Blink, I recommend either posting on their github issues tracker or if it is specific to your computer then asking on the [Julia discourse](https://discourse.julialang.org/) is another good place for information.

**Comment by lewisl** [comment]
Saw the fix for #220.  Will try that commit.

Thanks!
